README.md

```diff
@@ -55,7 +55,7 @@ npm run build
 ```json
 {
   "name": "my-library",
-  ["type": "commonjs",]
```

Comment:
My original intent was to communicate that `type: "commonjs"` is optional, and that `viteup` would consider both cases as commonjs packages.

I'm not sure it's actually relevant for consumers, this PR seems to indicate having non-syntactically correct examples potentially induces confusion.

With this context, do you think we should still go with the proposed change of this PR?

Reply:
This syntax was a bit confusing. IMO, given `"type": "commonjs"` is the default mode you can omit the line from the doc.